dev/how-to/deploy/tidb-in-kubernetes-prerequisites.md

```diff
@@ -0,0 +1,144 @@
+
+| Configuration Item | Value |
+| :--- | :--- |
+| net.core.somaxconn | 32768 |
+| vm.swappiness | 0 |
+| net.ipv4.tcp_syncookies | 1 |
+| net.ipv4.ip_forward | 1 |
+| fs.file-max | 1000000 |
+| fs.inotify.max_user_watches | 1048576 |
+| fs.inotify.max_user_instances | 1024 |
+| net.ipv4.neigh.default.gc_thresh1 | 80000 |
+| net.ipv4.neigh.default.gc_thresh2 | 90000 |
+| net.ipv4.neigh.default.gc_thresh3 | 100000 |
+| net.bridge.bridge-nf-call-iptables | 1 |
+| net.bridge.bridge-nf-call-arptables | 1 |
+| net.bridge.bridge-nf-call-ip6tables | 1 |
+
+When you set `net.bridge.bridge-nf-call-*` parameters, and if your option does not report errors, you can check whether this module is loaded by running the following command:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+lsmod|grep br_netfilter
+```
+
+If this module is not loaded, run the following command to load it:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+modprobe br_netfilter
+```
+
+You also need to disable swap on each deployed Kubernetes node by running:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+swapoff -a
+```
+
+To check whether swap is disabled:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+free -m
+```
+
+If the above command shows that the swap column is all `0`, then swap is disabled.
+
+In addition, to permanently disable swaps, remove all the swap-related entries in `/etc/fstab`.
+
+## Hardware and deployment requirements
+
++ 64-bit generic hardware server platform in the Intel x86-64 architecture and 10 Gigabit network card, which are the same as the server requirements for deploying a TiDB cluster using binary. For details, refer to [Hardware recommendations](/how-to/deploy/hardware-recommendations.md).
+
++ The server's disk, memory and CPU depends on the capacity planning of the cluster and the deployment topology. It is recommended to deploy three master nodes, three etcd nodes, and several worker nodes to ensure high availability of the online Kubernetes cluster.
+
+  Meanwhile, the master node often acts as a worker node (that is, load can also be scheduled to the master node) to make full use of resources. You can set [reserved resources](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/) by kubelet to ensure that the system processes on the machine and the core processes of Kubernetes have sufficient resources to run under high workloads. This ensures the stability of the entire system.
+
+The following text analyzes the deployment plan of three Kubernetes masters, three etcd and several worker nodes. To achieve a highly available deployment of multi-master nodes in Kubernetes, see [Kubernetes official documentation](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/high-availability/).
+
+## Kubernetes requirements for system resources
+
+- It is required on each machine to have a relatively large SAS disk (at least 1T) to store the data directories of Docker and kubelet.
+
+    > **Note:**
+  >
```

Comment:
```suggestion
  >
```
Please check the format of all **notes** and make sure they are indented with 4 spaces.